en/tutorials-and-examples/cms/authentication-plugin.rst

```diff
@@ -184,7 +184,7 @@ In your UsersController, add the following code::
         $this->Authentication->addUnauthenticatedActions(['login']);
     }
 
-    public function login() {
+    public function login()
```

Comment:
Shouldn't this brace stay?

Reply:
https://github.com/cakephp/docs/pull/6351

Reply:
This diff is removing the brace again though.